votor/src/voting_utils.rs

```diff
@@ -111,14 +115,19 @@ pub enum VoteError {
     SavedVoteHistoryError(#[from] VoteHistoryError),
 }
 
+pub(crate) struct BLSKeypairWithCompressedPubkey {
+    pub keypair: Arc<BLSKeypair>,
+    pub compressed_pubkey: [u8; BLS_PUBLIC_KEY_COMPRESSED_SIZE],
+}
```

Comment:
feels a little funky having this since the `compressed_pubkey` can be derived from the keypair. Should we have separate APIs and let callers decide how they want to access and mix/match things?

Reply:
Moving this to another PR later.